content/articles/servlet-request-and-servlet-collaboration/SevletRequest and ServletCallaboration.md

```diff
@@ -139,20 +138,21 @@ public class LoginAction extends HttpServlet {
 ```
 [Get project source code here]

```

Comment:
For this to work as expected, ensure this link description is on the same line as the link and no space between the ] and ( ie: `[github](www.github.com)`